draft-ietf-quic-multipath.md

```diff
@@ -758,9 +758,10 @@ is limited to a max value of 2^32-1.
 To calculate the nonce, a 96-bit path-and-packet-number is composed of the least
 significant 32 bits of the Path ID in network byte order,
 two zero bits, and the 62 bits of the reconstructed QUIC packet number in
-network byte order. If the IV is larger than 96 bits, the path-and-packet-number
+network byte order. The IV length is equal to the nonce length. If the IV is larger than 96 bits, the path-and-packet-number
 is left-padded with zeros to the size of the IV. The exclusive OR of the padded
-packet number and the IV forms the AEAD nonce.
+packet number and the IV forms the AEAD nonce. An AEAD algorithm where N_MIN
```

Comment:
N_MIN is not defined in this text. Would it be also okay to just say "nonce" instead? Or we need to add reference here.

Reply:
Yes, let's remove N_MIN